app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -193,7 +193,7 @@ if [ -f /etc/redhat-release ]; then
     fi
 
     subscription-manager register <%= '--force' if truthy?(@force) %> \
-      --org='<%= @organization.label %>' \
+      --org='<%= @organization.label if @organization %>' \
```

Comment:
Won't the registration fail without org anyway? If so, we probably want to omit the registration without the org, don't we?
```suggestion
      --org='<%= @organization&.label %>' \
```

Reply:
> Won't the registration fail without org anyway? 

Yes that's true but when you do the preview `@organization` variable is nil and preview action fails.

Reply:
Right, get it. It's little bit forced to alter the template just for preview, but these are small things so I'm ok :+1: